### DISCUSSION TITLE
[Bug] Service data is not displayed on the dashboard, while other data is displayed normally.

### DISCUSSION THREAD

**Discussion by hulkRfp** [apache/skywalking#12541] [Q&A]
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/skywalking/issues?q=is%3Aissue) and found no similar issues.

### Apache SkyWalking Component

OAP server (apache/skywalking)

### What happened

OAP server: 10.0.1
banyaandb: 0.6.1
After my service was running today, PHP's data suddenly stopped displaying on Dashboards. The data display of Visual Database and Visual Cache is normal.
At this point, I have to restart the OAP backend service. After restarting, it returned to normal and the data was not lost. I did not see any clear error message in the log.
This situation occurred twice within a week.

### What you expected to happen

I don't know.

### How to reproduce

I don't know.

### Are you willing to submit a pull request to fix on your own?

- [ ] Yes I am willing to submit a pull request on my own!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

**Comment by wu-sheng** [comment]
It is hard to tell. This may be something relative to data cycle management bug somehow. But it is hard to tell.

**Comment by wu-sheng** [comment]
We fixed some critical bugs in BanyanDB 0.6.1, and will do 0.7 release at the end of this month with SkyWalking 10.1.

**Comment by hanahmily** [comment]
It might relate to #12440
